microsoft-edge/devtools-guide-chromium/customize/theme.md

```diff
@@ -0,0 +1,78 @@
+---
+description: How to apply different color themes to Microsoft Edge DevTools.
+title: Apply color themes to Microsoft Edge DevTools
+author: MSEdgeTeam
+ms.author: msedgedevrel
```

Comment:
Was curious whether we can add feature owners the the author list to help users find the relevant point of contact.

Reply:
I'm not familiar yet with how this field is processed in the build and whether we can add more people to it. It'd be great if we could. I'll check.